File: projects/engineering.purdue.edu/scriptingwithobjects/swocode/chap13/FirstWindow.pl
#!/usr/bin/perl -w

###  FirstWindow.pl

use strict;                                                 
use Tk;                                                              #(A)

my $mw = MainWindow->new();                                          #(B)
$mw->title( "My First Window" );                                     #(C)
$mw->geometry( "200x150+50+100" );                                   #(D)
$mw->maxsize( 400, 500 );                                            #(E)
$mw->minsize( 100, 200 );                                            #(F)

my $label = $mw->Label( -text    =>    'hello',                      #(G)
                        -anchor  =>    's',                          #(H)
                        -relief  =>    'groove',                     #(I)
                        -width   =>    10,                           #(J)
                        -height  =>    3,                            #(K)
                      );
$label->pack( -side => 'top',                                        #(L)
              -padx => 10,                                           #(M)
              -pady => 5 );                                          #(N)

MainLoop();                                                          #(O)

File: projects/demos/engineering.purdue.edu/scriptingwithobjects/swocode/chap13/FirstWindow.py
#!/usr/bin/env python

###  FirstWindow.py

from Tkinter import *                                                #(A)

mw = Tk()                                                            #(B)
mw.title( "My First Window" )                                        #(C)
mw.geometry( "200x150+50+100" );                                     #(D)
mw.maxsize(400, 500)                                                 #(E)
mw.minsize(100, 200)                                                 #(F)

label = Label( mw,                                                   #(G)
               text="hello",                                         #(H)
               anchor  = 's',                                        #(I)
               relief  = 'groove',                                   #(J)
               width  = 10,                                          #(K)
               height = 3  )                                         #(L)

label.pack( side = 'top',                                            #(M)
            padx = 10,                                               #(N)
            pady = 5 )                                               #(O)

mainloop()                                                           #(P)
